blocks/core/ff_module/ff_module-response-recipient-list/_ff_module-response-recipient-list-renderer.js

```diff
@@ -10,7 +10,7 @@ var responses = [{
     },
     guid: "u47",
     label: "Sally Student",
-    latestEvent: {
```

Comment:
actually, these should still be latestEvent, but the inner object needs to be wrapped in `description`;
```
latestEvent:{
   description: {
        type: eventTypes.markAndGrade,
        sent: new Date()
    }
}
```